### PR TITLE
$Sheesh token

### PR DESCRIPTION
**Pull request by dturska12**
Add support for Sheesh token

### CODE REVIEW

File: packages/indexer/src/config/network.ts [hunk 1]
```diff
@@ -310,29 +310,14 @@ export const getNetworkSettings = (): NetworkSettings => {
             },
           ],
           [
-            "0xda9f05a3e133c2907e7173495022a936a3808d45",
```
Comment (nofir) [inline packages/indexer/src/config/network.ts:313]: Why were Nelkcoin and RugBank Token removed?

Comment (dturska12) [reply packages/indexer/src/config/network.ts:313]: When submitting the pr it said they were causing a conflict, can resubmit if need be. I must have must read what the issue was

Comment (nofir) [reply packages/indexer/src/config/network.ts:313]: yeah, just seems like a mistake during merge

Comment (dturska12) [reply packages/indexer/src/config/network.ts]: fixed and sumbitted, sorry about that last one

**Pull request closed without merging**